platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -1,61 +1,296 @@
-
-* Axe browser plugin checks on a subset of pages to ensure coverage
-* Zooming layouts to 400% on a subset of pages
-* Once-through keyboard navigation
-* 3 preferred screen reader and browser pairings. Pairings will be determined by analytics data and WebAIM’s [most recent screen reader survey](https://webaim.org/projects/screenreadersurvey8/#primary). The favored browser and screen reader pairings may evolve as Edge Chromium and Chrome assume more market share, but the current pairings include:
-  * JAWS on Windows7/10 using IE11 _(This is the VA 508 office's preferred test pairing)_
-  * NVDA on Windows 7/10 using Firefox
-  * VoiceOver on MacOS
-
-## Review New Interfaces and Patterns
-* Validate code early and often to help identify issues. This includes running axe checks and HTML validations, as well as basic keyboard and screen reader tests.
-* Request spot checks as needed before your staging review. Spot checks are meant to be collaborative sessions where interfaces or code can be discussed, reviewed, and tested. Spot checks can be a Slack conversation or informal video call.
-
-## Issue Gathering and Triage
-* Issues will be collected as issues in a new ZenHub epic. The accessibility specialist will assign this epic to the person who requested the staging review.
-* Issues will be triaged according to their severity. Some issues will be launch blockers. Others can be fixed post-launch. Teams should review the [accessibility defect severity rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md) if they have questions about issue severity or impact.
-* Accessibility specialists will work with stakeholders as issues are ready to be re-tested. When an issue has been resolved, the specialist or project manager should close it with a brief comment like _"Issue has been tested, and success criteria met. Moving to close."_
+
+# Accessibility staging review preparation
+
+Teams must conduct their own accessibility testing **before** scheduling a VSP Collaboration Cycle [Staging Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#staging-review). 
+
+Staging reviews test the [happy path](https://en.wikipedia.org/wiki/Happy_path) through the application or content page(s). This may not catch all accessibility issues, but will ensure basic functionality is accessible. (Accessibility specialists conduct a [more thorough pre-launch audit](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-request-prelaunch-review.md) when an application is relatively code-stable in production.)
+
+
+## Accessibility checklist for staging review
```

Comment:
Maybe even make this more specific "Accessibility checklist for team to complete before staging review" ?

Reply:
On second thought, maybe "Required accessibility checklist ..." ? To differentiate from recommended tests

Reply:
In my VSA checklist, we separated the Required from the Recommended. 

I find teams still aren't reading the first required item — reading. I suspect we're catching them at a time when they have so much to do. Perhaps we provide this information at on-boarding going forward?

Reply:
This new version removes the reading as a requirement in the first step and incorporates the key information from what they would have read into the description of the checks. However, I do like the idea of introducing this doc earlier in the collaboration cycle since the timing of the checks are such that teams should be incorporating it into their work from day 1.